pkg/sfu/streamtracker/streamtracker_packet_test.go

```diff
@@ -191,7 +191,7 @@ func TestStreamTracker(t *testing.T) {
 		require.Equal(t, uint32(2), callbackCalled.Load())
 
 		// first packet after reset
-		tracker.Observe(0, 20, 10, false, 0)
```

Comment:
Is it worth adding some tests for DD stream tracker?

Reply:
Done